.github/workflows/rust.yml

```
@@ -125,3 +125,38 @@
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Install dependencies
+        run: |
+          sudo apt-get update -y
+          sudo apt-get install -y librocksdb-dev libsnappy-dev libsqlite3-dev
+
+      - name: Install stable toolchain
+        uses: actions-rs/toolchain@v1
```

actionlint

the runner of "actions-rs/toolchain@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)